wagtail/admin/tests/test_admin_search.py

```diff
@@ -107,3 +109,78 @@ def test_search_other(self):
 
         self.assertNotIn("Pages", rendered)
         self.assertIn("My Search", rendered)
+
+
+class TestCase(BaseSearchAreaTestCase):
+    """Tests the comparison functions."""
+
+    def setUp(self):
+        self.search_area1 = SearchArea("Label 1", "/url1", order=100)
+        self.search_area2 = SearchArea("Label 2", "/url2", order=200)
+        self.search_area3 = SearchArea("Label 1", "/url3", order=300)
+
+    def test_eq(self):
+        self.assertEqual(
+            self.search_area1, self.search_area1
+        )  # Same object, should be equal
+        self.assertEqual(
+            self.search_area1, self.search_area1
+        )  # Same label and order, should be equal
```

Comment:
These two assertions are identical. Instead, we should have a different instance that has the same properties.

Also, could you please put the comment above each assertion instead, so it's formatted as two lines instead of three. Do the same for the rest of this PR too